src/routes/login/+page.svelte

```diff
@@ -29,48 +122,60 @@
                 <span>Email</span>
             </label>
             <input
-                class="input {emailIndicator}"
+                class="bg-surface-700 text-white border rounded-full transition-colors {emailIndicator} w-full focus:border-primary-500 focus:outline-none focus:ring-0"
```

Comment:
Das tut mir jetzt schon nen bisschen Leid :/, aber das geht deutlich einfacher durch die vorgeschlagenen Änderungen für die Indicator Status Klassen. Dadurch wird das `important` Attribut gesetzt und die default border überschrieben.
```suggestion
                class="input {emailIndicator}"
```